app/src/main/java/org/oppia/android/app/options/OptionsFragment.kt

```diff
@@ -25,6 +25,13 @@ class OptionsFragment : InjectableFragment() {
   lateinit var optionsFragmentPresenter: OptionsFragmentPresenter
 
   companion object {
+    const val KEY_READING_TEXT_SIZE_PREFERENCE_SUMMARY_VALUE =
+      "READING_TEXT_SIZE_PREFERENCE_SUMMARY_VALUE"
+    const val APP_LANGUAGE_PREFERENCE_SUMMARY_VALUE_EXTRA_KEY =
+      "AppLanguageActivity.app_language_preference_summary_value"
+    const val KEY_AUDIO_LANGUAGE_PREFERENCE_SUMMARY_VALUE =
```

Comment:
If you are introducing these value then please remove the previous consts mentioned at the top of the kotlin file.
Or alternate can be to not introduce these consts and use the values mentioned at top of kotlin file in your test file.

Reply:
I have removed the previous consts mentioned at the top of the test file. PTAL.

Reply:
Actually you have removed it from test file. But instead what we want it we should either use only 
```
const val MESSAGE_READING_TEXT_SIZE_ARGUMENT_KEY = "OptionsFragment.message_reading_text_size"
const val MESSAGE_APP_LANGUAGE_ARGUMENT_KEY = "OptionsFragment.message_app_language"
const val MESSAGE_AUDIO_LANGUAGE_ARGUMENT_KEY = "OptionsFragment.message_audio_language"
```
in tests and production code

or we should use only
```
const val KEY_READING_TEXT_SIZE_PREFERENCE_SUMMARY_VALUE =
      "READING_TEXT_SIZE_PREFERENCE_SUMMARY_VALUE"
    const val APP_LANGUAGE_PREFERENCE_SUMMARY_VALUE_EXTRA_KEY =
      "AppLanguageActivity.app_language_preference_summary_value"
    const val KEY_AUDIO_LANGUAGE_PREFERENCE_SUMMARY_VALUE =
      "AUDIO_LANGUAGE_PREFERENCE_SUMMARY_VALUE"
```


Currently you are using the above for the app layer code and the below for test cases which is incorrect. We should use only one of these sets in both code.

Reply:
Actually, we are passing the key `KEY_READING_TEXT_SIZE_PREFERENCE_SUMMARY_VALUE` while creating the `ReadingTextSizeActivity` intent. In the `OptionsFragment` test, we are checking that the correct intent has passed or not when the user clicks on the `RecyclerView` item, so we need the value of this key in the test file to match the intent passed, but the problem with accessing this key in the test file is that it has been declared as an `internal const` in the companion object of `ReadingTextSizeActivity` therefore the test fails in Bazel roboelectric because of the Test file not being able to access those internal consts, therefore I have to again declare them in `companion object` of `ReadingTextSizeFragment`. Please suggested an alternative approach to this if this is incorrect so that we can access the key in the test file.

Reply:
You can use the values of removing the `internal` in `ReadingTextSizeActivity` . Won't that work? @Sparsh1212 

Also the solution where you have suggested introducing them in `ReadingTextSizeFragment ` is incorrect because its not part of intent or activity so the testing would be incorrect.

Reply:
@rt4914  Yes removing `internal` in `ReadingTextSizeActivity` should work. Also, apologies about the alternative approach, I meant `OptionsFragment` not `ReadingTextSizeFragment`. Made a typo there. 
I have updated the changes. PTAL.
Also, all the CI checks are failing, I hope this isn't due to this PR.